server/routes/testutils/appSetup.ts

```diff
@@ -60,7 +59,7 @@ function appSetup(
 
   nunjucksSetup(app, testAppInfo)
   app.use((req, res, next) => {
-    if (!sessionData.booker && populateBooker) {
+    if (!sessionData.booker) {
```

Comment:
This change is just to a testing helper: `populateBooker` was a way to mock a user being logged in to One Login but without a booker reference. Now that we auto-create booker references the first time a user logs in this scenario is redundant. So removing from helper and some tests.